### PR TITLE
Yelp changed

### PR DESCRIPTION
**Pull request by ashalya**

### CODE REVIEW

File: yelp/yelp-connector/yelp-connector-1.0.0/src/main/resources/yelp_business/business.xml [hunk 1]
```diff
@@ -63,6 +63,13 @@
         <property name="uri.var.apiUri" expression="get-property('yelp.apiUri')"/>
         <property name="uri.var.signature" expression="get-property('yelp.oauth.signature')"/>
         <property name="uri.var.parameters" expression="get-property('yelp.parameters')"/>
+        <script language="js">
+            <![CDATA[
+                   var query = mc.getProperty('uri.var.signature');
+					var encodedQuery = encodeURIComponent(query);
+					mc.setProperty('uri.var.signature', encodedQuery);
```
Comment (kesavany) [inline yelp/yelp-connector/yelp-connector-1.0.0/src/main/resources/yelp_business/business.xml]: Format the code.

File: yelp/yelp-connector/yelp-connector-1.0.0/src/main/resources/yelp_search/search.xml [hunk 2]
```diff
@@ -200,6 +200,13 @@
         <property name="uri.var.signature" expression="get-property('yelp.oauth.signature')"/>
         <property name="uri.var.parameters" expression="get-property('yelp.parameters')"/>
         <property name="uri.var.apiUri" expression="get-property('yelp.apiUri')"/>
+        <script language="js">
+            <![CDATA[
+                   var query = mc.getProperty('uri.var.signature');
+					var encodedQuery = encodeURIComponent(query);
+					mc.setProperty('uri.var.signature', encodedQuery);
```
Comment (kesavany) [inline yelp/yelp-connector/yelp-connector-1.0.0/src/main/resources/yelp_search/search.xml]: Format the code.

**Comment by ashalya** [comment]
fixed in https://github.com/wso2/esb-connectors/pull/432

File: yelp/yelp-connector/yelp-connector-1.0.0/src/main/resources/yelp_business/business.xml (hunk 1 above)
Comment (ashalya) [inline yelp/yelp-connector/yelp-connector-1.0.0/src/main/resources/yelp_business/business.xml]: fixed in https://github.com/wso2/esb-connectors/pull/432

**Comment by ashalya** [comment]
All license header fixed in https://github.com/wso2/esb-connectors/pull/448

**Pull request merged**